comp/forwarder/defaultforwarder/go.mod

```diff
@@ -56,19 +56,19 @@ require (
 	github.com/DataDog/datadog-agent/comp/core/log/mock v0.0.0-00010101000000-000000000000
 	github.com/DataDog/datadog-agent/comp/core/status v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/config/mock v0.58.0-devel
-	github.com/DataDog/datadog-agent/pkg/config/model v0.56.0-rc.3
-	github.com/DataDog/datadog-agent/pkg/config/setup v0.56.0-rc.3
+	github.com/DataDog/datadog-agent/pkg/config/model v0.57.1
```

Comment:
Is it expected that it updates the version of some modules to v0.57.1 ?

Reply:
This is done automatically by `go mod tidy`.